docs/sources/fundamentals/dashboards-overview/index.md

```diff
@@ -0,0 +1,88 @@
+---
+aliases:
+  - ../basics/timeseries/
+description: Grafana Dashboards Overview
+keywords:
+  - grafana
+  - dashboards
+  - panel
+  - data source
+  - transform
+  - query
+title: Grafana Dashboards Overview
+weight: 300
+---
+
+# Grafana Dashboards Overview
+
+What is a dashboard? This term is used widely in the observability space, but what does it actually mean? The term is taken from automobiles where a dashboard provides access to the controls a driver uses to operate a vehicle. The same applies to digital dashboards, except that digital dashboards help us understand and drive systems. The purpose of this topic is to explain how Grafana dashboards work to make it easier for you to build your own.
+
+The following image illustrates a sample Grafana dashboard.
+
+{{< figure src="/media/docs/grafana/dashboards-overview/complex-dashboard-example.png" max-width="750px" caption="Example Grafana Dashboard" >}}
+
+A Grafana dashboard is a collection of data visualization panels. Each panel relies on a collection of components that  takes raw facts from a data source and shapes them into a beautiful visualization. The raw facts pass through three gates before being visualized, one of which is optional: a plugin, a query, and a transform.
+
+The following image illustrates each of the gates, and the following sections describe each element, how it’s used, and why it’s necessary. The journey of a visualization begins at the data source and works its way through to a Grafana dashboard.
+
+{{< figure src="/media/docs/grafana/dashboards-overview/dashboard-component-architecture.png" max-width="750px" caption="Dashboard component architecture" >}}
+
+## Data Source
+
+A data source is any entity that contains data. A data source can be a SQL database, Grafana Loki, Grafana Mimir, or a JSON-based API. A data source can even be a flat CSV file. A dashboard visualization starts with the data source that contains the data you want to see.
```

Comment:
```suggestion
A data source refers to any entity that consists of data. It can be a SQL database, Grafana Loki, Grafana Mimir, or a JSON-based API. It can even be a basic CSV file. The first step in creating a dashboard visualization is selecting the data source that contains the data you need.
```